apps/aeoracle/src/aeo_query_tx.erl

```diff
@@ -115,6 +116,10 @@ check(#oracle_query_tx{sender = SenderPubKey, nonce = Nonce,
         {error, Reason} -> {error, Reason}
     end.
 
+-spec accounts(query_tx()) -> [pubkey()].
+accounts(#oracle_query_tx{sender = SenderPubKey}) ->
```

Comment:
Should the oracle be part of the accounts (i.e. the oracle that gets the query?)

Reply:
Good question ...